<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mileage</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../logo.png</normaloff>../logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="milesVBox">
        <item>
         <widget class="QLabel" name="milesLabel">
          <property name="text">
           <string>Miles:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="milesEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="costVBox">
        <item>
         <widget class="QLabel" name="costLabel">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Cost (Â£):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="costEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="litresVBox">
        <item>
         <widget class="QLabel" name="litresLabel">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Litres:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="litresEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="btnLayout">
        <item>
         <widget class="QPushButton" name="combineButton">
          <property name="text">
           <string>Combine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="mileageList"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionClear_History"/>
   </widget>
   <addaction name="menu_About"/>
  </widget>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionClear_History">
   <property name="text">
    <string>Clear History</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
